engine/Dockerfile

```
@@ -1,4 +1,10 @@
 FROM python:3.11.3-slim-buster AS base
+
+# Create a group and user to run an app
+ENV APP_USER=appuser
+RUN groupadd -r -g 1000 ${APP_USER} && \
```

These flags are equivalent according to the [manpage](https://man7.org/linux/man-pages/man8/groupadd.8.html#:~:text=The%20groupadd%20command%20creates%20a,digits%2C%20underscores%2C%20or%20dashes.) for `groupadd`
```suggestion
RUN groupadd --system --gid 1000 ${APP_USER} && \
```
I would prefer the more verbose flags for readability